docs/data.json

```diff
@@ -2,9 +2,16 @@
     "project": {
         "name": "c2fo-ui-components",
         "description": "C2FO core ui components",
-        "version": "0.0.3"
+        "version": "1.0.0"
```

Comment:
1.0.0 is the first breaking change. This was a massive commit that changed the export architecture and thus is affecting the admin currently. Admin will need changes before updating from 0.0.3.